platforms/pf-robot-motion-control/motion_control.cpp

```diff
@@ -363,8 +367,8 @@ void pf_handle_target_pose(cogip::canpb::ReadBuffer &buffer)
         }
 
         // Target speed
-        target_speed.set_distance((platform_max_speed_linear_mm_per_period * target_pose.max_speed_ratio_linear()));
-        target_speed.set_angle((platform_max_speed_angular_deg_per_period * target_pose.max_speed_ratio_angular()));
+        target_speed.set_distance((platform_max_speed_linear_mm_per_period * target_pose.max_speed_ratio_linear()) / 100);
+        target_speed.set_angle((platform_max_speed_angular_deg_per_period * target_pose.max_speed_ratio_angular()) / 100);
```

Comment:
I'm not sure to understand the reason to divide by 100 here ? Is this a unit conversion?